test/TestMatchError.py

```diff
@@ -2,13 +2,12 @@
 
 import operator
 
+import pytest
```

Comment:
The problem here is the same as in other places: isort thinks that ansiblelint is third-party and puts it together with pytest.